webapp/channels/src/components/menu/menu.tsx

```diff
@@ -205,7 +212,7 @@ export function Menu(props: Props) {
                     id={props.menuButtonTooltip.id}
                     title={props.menuButtonTooltip.text}
                     placement={props?.menuButtonTooltip?.placement ?? 'top'}
-                    disabled={isMenuOpen}
+                    disabled={props.hideTooltipWhenDisabled ? props.menuButton.disabled || isMenuOpen : isMenuOpen}
```

Comment:
```suggestion
                    disabled={props.hideTooltipWhenDisabled ? props.menuButton.disabled  : isMenuOpen}
```
shouldnt this be just this?

Reply:
No, this will cause the button tooltip to continue showing up unless its disabled by props. Hiding the tooltip automatically on opening the manu is an existing behaviour which would be broken with this suggestion.